src/renderer/index.js

```diff
@@ -79,6 +82,9 @@ render(
 						<Route path="/backupPrivateKey" component={BackupPK} />
 						<Route path="/main" component={Main} />
 						<Route path="/unlockWallet" component={Unlock} />
+						<Route path="/enterTrezorPin" component={EnterPin} />
```

Comment:
please use lowercase letters with dashes, it is a url after all

Reply:
@sk91 it is not a web app so there is no real need to be a dashed separated string. We could but at this point there are a lot of routings using this format so I would not change that right now. Perhaps we can change all together later if needed. cheers